newt/imgprod/imgprod.go

```diff
@@ -312,10 +318,55 @@ func OptsFromTgtBldr(b *builder.TargetBuilder, ver image.ImageVersion,
 
 func ProduceAll(t *builder.TargetBuilder, ver image.ImageVersion,
 	sigKeys []sec.PrivSignKey, encKeyFilename string, encKeyIndex int,
-	hdrPad int, imagePad int) error {
+	hdrPad int, imagePad int, sectionString string) error {
+
+        elfPath := t.AppBuilder.AppElfPath()
+
+        cmdName := "arm-none-eabi-objdump"
```

Comment:
This seems like something that should be specified in the compiler package.

I see newt does this same thing in the `size` command, so I can't fault you for doing it here.  Maybe a future improvement to consider later.

Reply:
yeah good point.  I'll have to put that on the future todo list.

Reply:
go fmt comment addressed.  Pushed new set of changes incorporating that change.